dev-packages/node-integration-tests/utils/run-tests.ts

```diff
@@ -73,15 +85,35 @@ const workers = os.cpus().map(async (_, i) => {
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 Promise.all(workers).then(() => {
   console.log('-------------------');
-  console.log(`Successfully ran ${numTests} tests.`);
-  if (fails.length > 0) {
-    console.log('Not all tests succeeded:\n');
+
+  const failCount = fails.length;
+  const skipCount = skips.length;
+  const totalCount = numTests;
+  const successCount = numTests - failCount - skipCount;
+  const nonSkippedCount = totalCount - skipCount;
+
+  if (skips.length) {
+    console.log('\x1b[2m%s\x1b[0m', '\nSkipped tests:');
```

Comment:
theoretically, we could use `chalk` or similar libraries for the color bytes but I think this is also fine

Reply:
yeah, I thought about this as well - I think it is OK for now, if we want to do more with that we can always switch over to chalk!